docs/content.zh/docs/connectors/datastream/kinesis.md

```diff
@@ -1,75 +1,76 @@
 ---
-title: Kinesis
-weight: 5
-type: docs
-aliases:
-  - /zh/dev/connectors/kinesis.html
-  - /zh/apis/streaming/connectors/kinesis.html
+标题: Kinesis
+权重: 5
+类型: docs
+别名:
+  - /dev/connectors/kinesis.html
+  - /apis/streaming/connectors/kinesis.html
```

Comment:
These shouldn't be translated/changed, this is metadata used by our documentation tooling